Vagrantfile

```diff
@@ -43,6 +43,13 @@ Vagrant.configure(2) do |config|
 		# Customize the amount of memory on the VM:
 		vb.memory = "1024"
 	end
-	
+
+	$env_var_script = <<SCRIPT
+	echo # vagrant profile script > /etc/profile.d/vagrant.sh
```

Comment:
This won't do what you want. I would recommend putting the following in `bin/setup-vagrant.sh`
```sh
cat <<EOS > /etc/profile.d/vagrant.sh
# script here
EOS
```